rtf_vis_tool/src/Components/FrontendSummary.js

```diff
@@ -0,0 +1,63 @@
+/* Component to render Frontend Summary metrics. Specifically rotation sucesses, translation
+successes, pose successes, and correspondence inliers.
+
+Author: Adi Singh
+*/
+import React from "react";
+
+// Local Imports.
+import '../stylesheets/FrontendSummary.css'
+
+function FrontendSummary(props) {
+    /*
+    Args:
+        props.json.rotation.success_count (int): Number of successes from rotation averaging.
+        props.json.translation.success_count (int): Number of successes frmo translation averaging.
+        props.json.num_valid_entries (int): Number of valid image pair entries.
+        props.json.num_total_entries (int): Number of total image pair entries.
+        props.json.pose.success_count (int): Number of pose successes.
+        props.json.correspondence.all_inliers (int): Number of correspondence inliers.
+        
+    Returns:
+        A component showing frontend summary metrics after clicking the 'TwoViewEstimator' Plate.
+    */
+
+    return (
+        <div className="fs_container">
+            <div className="fs_sub_container">
```

Comment:
you use a combination of `_` and `-` for class names. I think you should stick with one convention

Reply:
good point. I'll standardize all my `className` tags (which correspond to CSS classes) to use `_`.